core/src/main/resources/lib/hudson/project/config-trigger_pl.properties

```diff
@@ -19,4 +19,4 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-Build\ Triggers=Wyzwalacze zadania
+Build\ Triggers=Wyzwalacze budowania
```

Comment:
I think old translation was better. 
Polish word which refers to `Build` is used here more like a `task`.
Jobs not always build stuff. 😉 
```suggestion
Build\ Triggers=Wyzwalacze zadania
```

Reply:
Agree with your but then we should decide if we want to have better translation OR consistent translation:
![obraz](https://user-images.githubusercontent.com/9612911/199564536-3f1f02d1-3168-4606-83f6-f5952734037a.png)
Build Steps -> Kroki zadania - IMO it sounds strange and we should keep original concept even this is not perfect

Reply:
Make sense, i wasnt able to find better translation.
Maybe `Składowe zadania`
when you have a math problem to solve there are steps you need to execute.
then you can refer to them as `Kroki zadania`

Reply:
Well, if you ask your dev team, tester and business consultant (all of then use Jenkins) about "Składowe zadania" without context and 4 of 5 give you good answer what is "Składowe zadanie" we can go for it and change it. But I believe they wont' give you that answer :)
Also mind that they know what is build because they have already used this word for TeamCity, Travis and other tools. 
_\- Can you start the build Mike?_ (sure he will and he knows that you are talking about CI/CD)
_\- Can you start the task Tom?_ (what task, what tool? probably JIRA subtask)

We should use same names no matter what language we speak. That helps us to understand each other :)

Reply:
Ok, i'm sold 😉